### DISCUSSION TITLE
Is there a split button component?

### DISCUSSION THREAD

**Discussion by Ajay100000** [microsoft/fluentui-blazor#1780] [General]
Fluent UI has a split button, for example here: https://github.com/microsoft/fluentui/issues/12307

Can Blazor Fluent UI have a split button as well, or have I missed it in the documentation?

Thanks.

**Comment by dvoituron** [comment]
You have the [FluentMenuButton](https://www.fluentui-blazor.net/MenuButton). It's not exactly the same thing, but it's similar.

**Comment by Ajay100000** [comment]
Ok thanks I will take a look when I am working on my side project. This is a very good set of controls for Blazor. Well done, it is appreciated.

**Comment by Ajay100000** [comment]
It is not quite the same, if you get time to implement the split button then that would be great.

**Reply by vnbaaij** [reply]
Hi,

It is not our aim to mimic the Fluent React library 100%. But, as the SpinButton is part of what is described in the general [Fluent 2 Design System](https://fluent2.microsoft.design/components/web/react/button/usage), there is a good change this will beimplemented in the future. We just don't have a timeline for it yet.
